WTWSMS/common/nicknames/00_nicknames.txt

```diff
@@ -1925,8 +1949,10 @@ nick_the_wolf = {
 #################################
 nick_the_hammer = {
 	allow = {
-		trait = brave
-		trait = champion
+		OR = {
+			trait = brave
+			trait = champion
+		}
```

Comment:
Why? The Nickname is never going to be given if both of those Traits exist. We need variety.